### PR TITLE
Adds a new command 'ontologies' with a single sub-command 'import' for loading data

### PR DESCRIPTION
**Pull request by tdstein**

### CODE REVIEW

**Comment by tdstein** [comment]
```
head ontology.json
[
  {
    "equivalence": "subsumes",
    "source": {
      "type": "coding",
      "system": "http://hl7.org/fhir/sid/icd-10-cm",
      "display": "Malignant neoplasm of head of pancreas",
      "code": "C25.0"
    },
    "target": {
```

```
 cat ontology.json | lo ontologies import cccdf419-ac83-4b7e-aa2d-70702d43297c
```

File: lib/cmds/ontologies_cmds/import.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  const project = argv.project;
+  const path = `/v1/terminology/projects/${project}/relationships/`;
+
+  const pipeline = chain([
+    stdin(),
+    parser(),
+    streamValues(),
+    async (data) => {
+      const value = data.value;
+      for (let i = 0; i < value.length; i++) {
```
Comment (markdlv) [inline lib/cmds/ontologies_cmds/import.js]: Since you're not using the index, seems like you could do a `for (const payload of value)` here.

**Comment by mschroering** [comment]
After you merge the PR, the current release process is manual, in that you need to update Changelog.md from `master`, and run `npm version ...` to bump the version and apply the version tag.  Once you push these changes (with the version tag) up to master (no PR required), the Travis build will kick off.

**Comment by tdstein** [comment]
I will be able to release this today or tomorrow. Please review once more when you have a moment.

**Comment by tdstein** [comment]
@mschroering - I need access rights to push to master.

**Pull request merged**